applications/web/templates/ingress.yaml

```
@@ -59,38 +64,37 @@ spec:
   {{- end }}
 
   rules:
-    {{- range .Values.ingress.hosts }}
-      - host: {{ . }}
-        http:
-          paths:
-            {{ if gt (len $customPaths) 0 }}
-            {{- range $customPaths }}
-            - path: {{ .path }}
-              backend:
-                serviceName: {{ .serviceName }}
-                servicePort: {{ .servicePort }}
-            {{- end }}
-            {{ else }}
-            - backend:
-                serviceName: {{ $fullName }}
-                servicePort: {{ $svcPort }}
-            {{ end }}
-    {{- end }}
-    {{- range .Values.ingress.porter_hosts }}
+    {{- range $allHosts }}
       - host: {{ . }}
         http:
           paths:
             {{ if gt (len $customPaths) 0 }}
             {{- range $customPaths }}
             - path: {{ .path }}
+              pathType: Prefix
```

```suggestion
              {{- if $.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
              pathType: ImplementationSpecific
              {{- end }}
```

I believe that `ImplementationSpecific` matches the previous behavior more closely than `Prefix`, and we only want to add `pathType` if the user is on `networking.k8s.io/v1`. Though I'm not sure about `networking.k8s.io/v1beta1`.